codespell_lib/data/dictionary.txt

```diff
@@ -23663,6 +23685,7 @@ officeally->officially
 officeals->officials
 officealy->officially
 officialy->officially
+offlaod->offload
```

Comment:
`offlaoded`?

Reply:
Yup, also added `offlaoding->offloading`.

Hmm, so [Merriam-Webster](https://www.merriam-webster.com/dictionary/off-load) prefers a hyphen, but [Collins](https://www.collinsdictionary.com/dictionary/english/off-load) says that spelling is a British variant. One solution is to just correct both misspellings, i.e. to add both `offlaod->offload` and `off-laod->off-load` (and so on). So that's what I did.